encoding/protobuf/testdata/istio.io/api/other/explicit_names.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
```

Comment:
can you name this `qualified_references` or `full_references` for clarity?

also, should this work for nested references too, like `current.package.name.SomeMessage.SomeSubMessage`?